src/vs/platform/actions/common/menuService.ts

```diff
@@ -410,7 +411,13 @@ function createMenuHide(menu: MenuId, command: ICommandAction | ISubmenuItem, st
 	const toggle = toAction({
 		id: `toggle/${menu.id}/${id}`,
 		label: title,
-		get checked() { return !states.isHidden(menu, id); },
+		get checked() {
+			// Invert the meaning of the isHidden state when the default is hidden
```

Comment:
instead of inverting which I think could be confusing down the line, can we just pass a default value to the isHidden function and have that returned if not stored in the state.